### PR TITLE
ISSUE #3173 - Viewer facelift - Issues details tab (sequences)

### LINKED ISSUE #3173: Viewer facelift - Issue details tab (sequences)
### Description
For the issues we need to style every tab. This one is for sequences

### Goals
- [ ] As a user I want to change issues sequences properties and have it styled in the v5 style

### Tasks
- [ ] Style the actual date picker (consult with George before starting this)
- [ ] Style blank state
- [ ] Style the input box for the picker  

### Notes:
We are not changing the button icons for now

### Related Resources
[MockUp](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=136544%3A177468)

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3173

#### Description
- adds v5 styling to the 4d(sequences) tab in the viewer
- adds placeholder text to datetime selectors
- cleans up some styling in the calendar/clock modal

#### Test cases
- Look at sequences for existing problem
- look at 4d tab when creating a sequence

### CODE REVIEW

**Comment by Amantini1997** [comment]
image
I think you should add the `at` between the date and the time and move the icons closer to the side

**Comment by Amantini1997** [comment]
The icons for the start and end data textfield get messed up when the scrollbar is rendered.
image

**Comment by Amantini1997** [comment]
The icons for the date textfield overlay the text when the scrollbar is render
image

I believe this could be because the styling that should be applied to the texfield-container is instead applied to its content. Since the position of the icons is relative to that container, this creates the problem.

**Comment by george-hadfield** [comment]
### Start and end time alignment is off
image

**Comment by george-hadfield** [comment]
### Buttons in time and date modal need to replace text links

image

**Comment by george-hadfield** [comment]
@Amantini1997 this is now ready for changes again

File: frontend/src/v5/ui/v4Adapter/overrides/properties.overrides.ts [hunk 1]
```diff
@@ -67,8 +67,8 @@ export default css`
 
 	// Drop-down inputs
 	${StyledFormControl} {
-		label { left: -14px; }
-		.MuiInput-root svg { margin-top: 23px; }
+		/* label { left: -14px; } */
```
Comment (Amantini1997) [inline frontend/src/v5/ui/v4Adapter/overrides/properties.overrides.ts:70]: If you don't need it anymore, could you please remove this comment?

Comment (The-Daniel) [reply frontend/src/v5/ui/v4Adapter/overrides/properties.overrides.ts:70]: oops

**Pull request merged**